main.cc

```diff
@@ -1591,6 +1591,8 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 cdc.stop().get();
             });
 
+            supervisor::notify("starting storage service", true);
```

Comment:
This place doesn't start storage service here in any means

Reply:
@xemul i agree that the message is wrong and misleading. but the goal of this change is to partially **revert** the change to address the regression. not to have a "correct" fix. we can always fix the message and / or the place where the statement is located.